proto/rill/runtime/v1/api.proto

```diff
@@ -15,6 +15,10 @@ service RuntimeService {
     option (google.api.http) = {get: "/v1/ping"};
   }
 
+  rpc Healthz(HealthzRequest) returns (HealthzResponse) {
+    option (google.api.http) = {get: "/v1/healthz"};
+  }
```

Comment:
Could it be called just `Health`?

Reply:
It seems like a convention that google [follows](https://stackoverflow.com/questions/43380939/where-does-the-convention-of-using-healthz-for-application-health-checks-come-f).
I can change it to `Health` as well.